### ISSUE TITLE
 train_network.py: error: unrecognized arguments: --deepspeed --zero_stage=2 --offload_optimizer_device=cpu

### ISSUE THREAD

**Issue by lianshuailong0715** [kohya-ss/sd-scripts#1262]
1.i have install deepspeed with pip
2.this is my train script:
accelerate launch --num_cpu_threads_per_process=2 train_network.py \
    --pretrained_model_name_or_path="xxx" \
    --train_data_dir="xxx" \
    --caption_extension='.txt' \
    --resolution="840,600" \
    --tokenizer_cache_dir="xxx" \
    --logging_dir="xxx" \
    --output_dir="xxx" \
    --save_model_as="safetensors" \
    --output_name="xxx" \
    --network_module=networks.lora \
    --network_alpha=64 \
    --network_dim=128 \
    --lr_scheduler="cosine" \
    --learning_rate=1e-5 \
    --lr_warmup_steps=1000 \
    --lr_scheduler_num_cycles=20 \
    --train_batch_size=1 \
    --max_train_epochs=20 \
    --save_precision='fp16' \
    --save_every_n_epochs=1 \
    --seed=1024 \
    --optimizer_type="AdamW8bit" \
    --deepspeed \
    --zero_stage=2 \
    --offload_optimizer_device="cpu" \

**Comment by BootsofLagrangian** [comment]
Did you update this repo? try script after `git pull`.
